lib/Db/Face.php

```diff
@@ -173,9 +183,13 @@ public function setDescriptor($descriptor) {
 		$this->markFieldUpdated('descriptor');
 	}
 
-	public function getCreationTime(): string {
-		// Deck app have special handling for MySQL here:
+	public function getCreationTime(bool $isoFormat = false): string {
+		// Same special handling for MySQL as in Deck app
 		// https://github.com/nextcloud/deck/blob/139b38ca1df0ff17792eb218cc8ba7e3b04b4e51/lib/Db/Card.php#L84
+		//
+		if (!$isoFormat && $this->databaseType === 'mysql') {
+			return $this->creationTime->format('Y-m-d H:i:s');
```

Comment:
Why not always convert to this full format?.
I'm looking at all the Nextcloud repositories, and only deck has this problem, and the rest, use a full formating.

Reply:
Because we are 1) using DateTime and not plain timestamp, and 2) we use proper getter/setters that need to joggle between string and DateTimer.

I think, if we go with timestamps, that we might be able to completely make this redundant. If you want, I can try doing that, but that would be backward incompatible change (and not sure if we need upgrade step for something not released?). But, this is bigger change, I would need sign-off from you that you would accept that as a change

Reply:
> I can try doing that, but that would be backward incompatible change (and not sure if we need upgrade step for something not released?)

That's exactly why I suggest it - the time to make incompatible changes is now, that we do not publish it yet.. :sweat_smile:

If you see it well, and you think it will not bring problems in the future, I approve it.

P.s: I know it will not bring problems, I would just prefer that it works the same with all the databases, and (being paranoid. haha.) I worry about the performance of the additional check. :sweat_smile:

Reply:
So, I think I fixed "original" problem, take a look at https://github.com/matiasdelellis/facerecognition/pull/126. If we merge that, this one is not needed!

Reply:
Ok..
Merge the branch #126 first, and then rebase it. :smile: